ethcore/light/src/client/header_chain.rs

```diff
@@ -524,7 +524,16 @@ impl HeaderChain {
 											None
 										}
 										Ok(None) => panic!("stored candidates always have corresponding headers; qed"),
-										Ok(Some(header)) => Some((epoch_transition, ::rlp::decode(&header))),
+										Ok(Some(header)) => {
+											match ::rlp::decode(&header) {
```

Comment:
this value was fetched from the database. It should never be invalid, therefore we should use `expect` here